src/components/containers/container.js

```diff
@@ -19,13 +19,17 @@ const Container = styled(Box)`
     @media ${device.laptopL} {
         width: 84%;
     }
+    @media ${device.laptopM} {
+        flex-direction: ${(props) => (props.laptop_direction ? props.laptop_direction : '')};
```

Comment:
```suggestion
        flex-direction: ${(props) => (props.laptop_direction)};
```
There's no need to add fallback for this as invalid css will be removed during build time

Reply:
okay